cmd/harmony/main.go

```diff
@@ -135,6 +139,15 @@ var (
 )
 
 func initSetup() {
+	switch *networkType {
+	case "mainnet":
+		config.Network = config.Mainnet
+	case "testnet":
+		config.Network = config.Testnet
+	case "devnet":
+		config.Network = config.Devnet
+	}
```

Comment:
return error and exit if the *networkType is not valid.

Reply:
Done.